ni_measurementlink_generator/ni_measurementlink_generator/template.py

```diff
@@ -146,6 +166,12 @@ def create_measurement(
 
     directory_out_path.mkdir(exist_ok=True, parents=True)
 
+    if annotations_description is None:
+        annotations_description = ""
+    if collection is None:
+        collection = ""
+    annotation_tags = json.dumps(tags)
```

Comment:
I think conversion to JSON belongs in the template, not the command line parser.

Reply:
Is this the command line parser? I am just manipulating data before cre_file the same way other stuff is being built on top.

Reply:
Pass the list into the template. Let the template handle presentation in JSON or whatever format it wants.